auditbeat/module/auditd/_meta/fields.yml

```diff
@@ -92,9 +92,6 @@
     type: group
     description: Process attributes.
     fields:
-    - name: exe
-      type: keyword
-      description: Absolute path of the executable.
```

Comment:
Instead of deleting, shouldn't this become an alias?

Reply:
Strange. I see the doc files aliasing to the new field. But where's the definition coming from? :-)

Reply:
No, as the alias already exists in libbeat, so it also applies to auditbeat.

Reply:
Never mind 🙄